### PR TITLE
pseudo: script for creating psuedo modules

### PR DESCRIPTION
**Pull request by pwittrock**
Fork the k8s.io/api, k8s.io/apimachinery and k8s.io/client-go deps so the kustomize code
can be vendored by kubernetes/kubernetes.

### CODE REVIEW

File: hack/fork-packages.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+set -o xtrace
+
+# fetch the packages if they don't exist
+for item in api apimachinery client-go
+do
+  if [ ! -d "forked/$item" ]; then
+    git clone https://github.com/kubernetes/$item forked/$item
```
Comment (Liujingfang1) [inline hack/fork-packages.sh]: Do we need to checkout a specific version for the those packages?

Comment (pwittrock) [reply hack/fork-packages.sh]: Not today

Comment (pwittrock) [reply hack/fork-packages.sh]: As a follow up, would you pin each of these deps to a specific version, and document in the README.md which version they were pinned to?

**Comment by monopole** [comment]
This is great Phil, thanks much!
/lgtm

**Comment by monopole** [comment]
@droot i get to keep my dollar, unfortunately

**Pull request merged**